.github/workflows/lint.yml

```
@@ -70,6 +70,30 @@ jobs:
             fi
           ' sh {} +
 
+  lint-image:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Lint image files
+        run: |
+          find "$FOLDER" -regextype posix-egrep -regex '.+\.(jpg|jpeg|png|webp)' -exec sh -c '
```

Nice! Maybe we can add `ico` and `svg` files

```suggestion
          find "$FOLDER" -regextype posix-egrep -regex '.+\.(ico|jpg|jpeg|png|svg|webp)' -exec sh -c '
```